newapi/ooniapi/citizenlab.py

```diff
@@ -411,16 +409,17 @@ def check_url(url):
         raise BadURL()
 
 
-def validate_entry(entry: List[str]) -> None:
-    url, category_code, category_desc, date_str, user, notes = entry
-    check_url(url)
-    if category_code not in CATEGORY_CODES:
+def validate_entry(entry: Dict[str, str]) -> None:
+    keys = ["category_code", "date_added", "notes", "url", "user"]
```

Comment:
They `user` key should be called `source`

Reply:
Commented out for now